ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -404,24 +402,11 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               '' : this.props.workspace.researchPurpose.diseaseOfFocus}
         }});
         if (this.isMode(WorkspaceEditMode.Duplicate)) {
-          this.setState({workspace: {
-            ...this.props.workspace,
-              // Replace potential nulls with empty string or empty array
-            researchPurpose: {
-              ...this.props.workspace.researchPurpose,
-              populationDetails: !this.props.workspace.researchPurpose.populationDetails ?
-                [] : this.props.workspace.researchPurpose.populationDetails,
-              diseaseOfFocus: !this.props.workspace.researchPurpose.diseaseOfFocus ?
-                '' : this.props.workspace.researchPurpose.diseaseOfFocus}
-          }});
-          if (this.isMode(WorkspaceEditMode.Duplicate)) {
-            this.setState({workspace: {
-              ...this.props.workspace,
```

Comment:
This is already set above.  Re-setting it here could overwrite values, so I removed it.